### PR TITLE
ROX-16723: add a job for operator tests on GKE

### PR DESCRIPTION
**Pull request by porridge**
I'm assuming this has no effect until invoked explicitly with `/test gke-operator-e2e-tests`, right?
And when invoked it will fail because there is no `stackrox/stackrox` counterpart for now? (This should be fine for the time being, while I work on that part.)

### CODE REVIEW

**Comment by porridge** [comment]
So this part of the failure is expected...

```
ERROR: There is no job script for gke-operator-e2e-tests
INFO: Wed Apr 26 09:29:47 UTC 2023: Executing a general purpose exit trap for CI
Exit code is: 1
```
... because there is no script yet.

But I don't think this one is, any clues @gavin-stackrox ?
```
INFO: Wed Apr 26 09:29:47 UTC 2023: Finalizing a job record for this test run
/go/src/github.com/stackrox/stackrox/scripts/ci/metrics.sh: line 145: METRICS_JOB_ID: unbound variable
```

**Comment by porridge** [comment]
So, can I please get and LGTM for this please? @gavin-stackrox

File: ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml [hunk 1]
```diff
@@ -232,6 +232,63 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )gke-nongroovy-e2e-tests,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (gavin-stackrox) [inline ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml:236]:
```suggestion
    always_run: false
```

**Review by gavin-stackrox — CHANGES_REQUESTED** [review]
the job needs a manual change to not always run.

**Review by gavin-stackrox — APPROVED** [review]
/lgtm

**Pull request merged**